tensorflow_probability/examples/jupyter_notebooks/Linear_Mixed_Effects_Models.ipynb

```diff
@@ -625,14 +629,16 @@
        " <tf.Variable 'intercept:0' shape=() dtype=float32, numpy=0.0>)"
       ]
      },
-     "execution_count": 0,
-     "metadata": {
-      "tags": []
-     },
+     "execution_count": 11,
+     "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
+    "# lmm_train is of type JointDistributionSequential which implictly inherits from tf.Module\n",
+    "# and since in above cell we attached the LinearMixedEffectModel Module to it you would\n",
```

Comment:
missing: "in _the_ above cell"

also `back_ticks` around `LinearMixedEffectModel` and `tf.Module` (and some inline code below)

Reply:
ok